digital_land/expectations/checkpoints/converted_resource.py

```diff
@@ -5,9 +5,82 @@
 # a checkpoint represents the moment in the process where we tell it the
 # type of data it is validating and where the data is
 # the primary different between checkpoints is how it loads expectations (i.e. where that are loaded from)
+from pathlib import Path
+import csv
 from .base import BaseCheckpoint
 
 
-class CovertedResourceCheckpoint(BaseCheckpoint):
-    def load():
-        pass
+class ConvertedResourceCheckpoint(BaseCheckpoint):
+    def __init__(self, data_path):
+        super().__init__("converted_resource", data_path)
+        self.csv_path = Path(data_path)
+
+    def load(self):
+        self.expectations = [
+            {
+                "function": self.check_for_duplicate_references,
```

Comment:
The function should be in the expectation_functions folder instead of a method on the class. Also see the loading function for the dataset checkpoint it dates three different lists in depending on the dataset and the dataset typology